cmd/control/control.go

```diff
@@ -1,67 +1,61 @@
 package main
 
 import (
-	"flag"
 	"fmt"
 	"log/slog"
 	"net/http"
 	"os"
 
+	"github.com/gpuctl/gpuctl/internal/config"
 	"github.com/gpuctl/gpuctl/internal/database"
 	"github.com/gpuctl/gpuctl/internal/database/postgres"
 	"github.com/gpuctl/gpuctl/internal/groundstation"
 	"github.com/gpuctl/gpuctl/internal/webapi"
 )
 
 func main() {
-
-	inMemDb := flag.Bool("inmemdb", false, "Use an transient, in-memory database")
-	postgresDb := flag.Bool("postgres", false, "Use a posgresql database from GPU_DB_URL")
-
-	flag.Parse()
-
 	log := slog.Default()
 	log.Info("Starting control server")
 
-	db, err := initialiseDatabase(inMemDb, postgresDb)
+	conf, err := config.GetServerConfiguration("control.conf")
```

Comment:
Why `.conf` and not `.toml`? The latter is clearer

Reply:
That is true. It started as one `config.toml` which needed to be two, so I pushed the `config` down into the extension, but `control.toml` is probably distinguishing enough

Reply:
Fixed in 7e12fd7

Reply:
Fixed in 762600c